build-parent/pom.xml

```diff
@@ -0,0 +1,224 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>event-connect</artifactId>
+    <groupId>com.redhat.developer</groupId>
+    <version>0.0.1</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>build-parent</artifactId>
+  <name>Event Connect build BOM</name>
+  <description>Event Connect Dependency Management Project BOM</description>
+
+  <properties>
+
+    <maven.compiler.release>8</maven.compiler.release>
+    <version.maven>3.6.2</version.maven>
+    <version.jdk>11</version.jdk>
+
+    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <maven.compiler.parameters>true</maven.compiler.parameters>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+    <quarkus.platform.version>2.1.3.Final</quarkus.platform.version>
+    <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
+    <version.compiler.plugin>3.8.1</version.compiler.plugin>
+    <version.surefire.plugin>2.22.2</version.surefire.plugin> <!-- minimum required by JUnit 5 -->
+    <version.install.plugin>2.5.2</version.install.plugin>
+    <kogito.version>1.10.0.Final</kogito.version>
```

Comment:
As far as I can see this version is only used to download the code formatting (while everything else is via quarkus platform bom) so please rename the property to avoid misunderstanding. What about using something like `kogito-formatter.version` or `kogito-codestyle.version`?